networkx/algorithms/tests/test_simple_paths.py

```diff
@@ -85,32 +85,32 @@ def test_multidigraph(self):
 def test_all_simple_paths():
     G = nx.path_graph(4)
     paths = nx.all_simple_paths(G,0,3)
-    assert_equal(list(list(p) for p in paths),[[0,1,2,3]])
```

Comment:
We don't want to use sets here (testing simple paths) because the ordering matters (paths).   There should, in this case, only be one simple path.